32_workflow-make-github-repo-browsable.Rmd

```diff
@@ -165,13 +165,21 @@ Hopefully we are moving towards a world where you can have "web friendly" and "v
   
 ## Other document formats
 
-You may also have a document you want others to be able to browse and interact with, but it is not in the markdown format. Fortunately, the open-source Pandoc program, written by John MacFarlane, allows you to convert a range of formats into markdown, including the widely used docx format.
+You may also have a document you want others to be able to browse and interact with, but it is not in the markdown format. Fortunately, the open-source Pandoc program, written by John MacFarlane, allows you to convert a range of formats into markdown, including the widely used `.docx` format.
 
-When you click the Knit button in Rstudio it is actually Pandoc which performs the final conversion to HTML or word (docx) formats. If you are willing to use the command-line, you can perform the opposite conversion (eg docx to markdown), commonly retaining features such as headings, tables, equations and even figures.
+When you click the Knit button in RStudio it is actually Pandoc which performs the final conversion to HTML or Microsoft Word (`.docx`) formats. If you are willing to use the command-line, you can perform the opposite conversion (eg `.docx` to `.md`), commonly retaining features such as headings, tables, equations and even figures.
 
-As some boilerplate, running in Windows PowerShell `pandoc --extract-media .\media  -f docx .\example.docx -t markdown_github -o example_image.md` converts a word document called example.docx to markdown, and extracts the images into a directory which corresponds to a filepath in the newly created example.md document. A full list of supported formats and example code for conversions are available at https://pandoc.org/.
+As some boilerplate, running in Windows PowerShell `pandoc --extract-media .\media  -f docx .\example.docx -t markdown_github -o example_image.md` converts a word document called `example.docx` to markdown, and extracts the images into a directory which corresponds to a filepath in the newly created `example.md` document. A full list of supported formats and example code for conversions are available at https://pandoc.org/.
 
-For simpler conversions between the different markdown flavours supported by Pandoc (mmd, php_extra and strict), you can convert these to Github using the Rstudio gui by simply renaming their extension to .Rmd.
+You can also perform simple conversions to GitHub-flavored mardkown from different markdown flavours (Pandoc support `markdown_mmd`, `markdown_php_extra` and `markdown_strict`) from within RStudio. To do so you need to rename the file by chaning the extension (eg from `foo.mmd` to `foo.Rmd`), then open the renamed file in RStudio and add the following text to the top of the document.
```

Comment:
mardkown --> markdown
Pandoc supports <-- add the "s"
chaning --> changing
foo.mmd --> foo.md